hr_employee_seniority/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# copyright 2013 Michael Telahun Makonnen <mmakonnen@gmail.com>
+# copyright 2017 Denis Leemann, Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+{
+    'name': 'Employee Seniority',
+    'summary': 'Keep Track of Length of Employment',
+    'version': '11.0.1.0.0',
+    'category': 'Human Resources',
+    'author': "Michael Telahun Makonnen <mmakonnen@gmail.com>, "
+              "Camptocamp SA, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://miketelahun.wordpress.com',
```

Comment:
URL should be `https://github.com/OCA/hr`

Reply:
Changed!